studio/components/layouts/ProjectLayout/LayoutHeader/HelpPopover.tsx

```diff
@@ -49,6 +50,13 @@ const HelpPopover: FC<Props> = () => {
                   </Button>
                 </a>
               </Link>
+              <Link href="https://status.supabase.com/">
+                <a>
+                  <Button type="secondary" size="tiny" icon={<IconActivity />}>
+                    Status
```

Comment:
maybe rename to "Supabase Status"

Reply:
To get the buttons all aligned I've changed the 'Contact support team' to 'Contact support' and also the spacing has been made smaller:

<img width="395" alt="Screenshot 2022-08-08 at 11 24 42" src="https://user-images.githubusercontent.com/22655069/183397513-2cc4dace-f0f5-4a6a-b0e7-521f8f1667d5.png">

@MildTomato - Shall I push the commit?

Reply:
Looks good @Hallidayo, give that a push!

Reply:
@saltcod - Cool just pushed a commit.